projects/igniteui-angular/src/lib/data-operations/filtering-condition.ts

```diff
@@ -22,12 +22,17 @@ export class IgxFilteringOperand {
         }, {
             name: 'in',
             isUnary: false,
+            iconName: 'filter_in',
+            hidden: true,
+            logic: (target: any, searchVal: Set<any>) => this.findValueInSet(target, searchVal)
+        }, {
+            name: 'inQuery',
+            isUnary: false,
             isNestedQuery: true,
             iconName: 'in',
             logic: (target: any, searchVal: Set<any>) => this.findValueInSet(target, searchVal)
```

Comment:
I realize `logic` is probably required, but not sure this is even valid; Just FYI probably should be looked at and possibly replaced with blanket true or someting.